packages/components/src/components/Switch/switch.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { Switch } from '.';
+
+export default {
+  title: 'components/Switch',
+  component: Switch,
+};
+
+export const Basic = () => <Switch />;
+
+export const defaultOn = () => <Switch defaultOn />;
+
+/* eslint-disable no-console */
+export const onChange = () => <Switch onChange={action('changed')} />;
```

Comment:
This makes the console log be on the addons tab

Reply:
amazing,thanks